internal/version/version.go

```diff
@@ -84,11 +94,14 @@ type Version struct {
 	URL     string `json:"url"`
 }
 
-// SemVerRegex is the regular expression used to parse a semantic version.
-// https://github.com/Masterminds/semver/blob/master/LICENSE.txt
-const SemVerRegex string = `v?([0-9]+)\.([0-9]+)\.([0-9]+)`
+// Rollout represents the state of a version rollout.
+type Rollout struct {
+	Seed   rolloutSeed `json:"seed"`
+	Cursor int32       `json:"cursor"`
```

Comment:
The `Cursor` should be a 32-byte (not 32-bit) too, i.e. `[32]byte`

Reply:
Yea, that makes sense